includes/widgets/image.php

```diff
@@ -447,11 +458,25 @@ protected function register_controls() {
 			]
 		);
 
+		$this->add_group_control(
+			Group_Control_Box_Shadow::get_type(),
+			[
+				'name' => 'image_box_shadow_hover',
```

Comment:
The old control has a prefix by mistake. So, a new control don't need it.

```suggestion
				'name' => 'shadow_hover',
```

Reply:
@rami-elementor Updated